### DISCUSSION TITLE
I get the following error msg while trying to login to Rosie. I am a first time user..

### DISCUSSION THREAD

**Discussion by Kutub-create** [RosettaCommons/rosetta#199] [ROSIE] [answered]
"Your GitHub account does not have verified academic email address! Please visit https://github.com/settings/emails and setup it! If your institution _is academic_ please contact our support forum at [https://github.com/RosettaCommons/rosetta/discussions/categories/rosie] and provide your institution name and domain address." 
Although my academic email id is verified..

**Comment by roccomoretti** [comment]
The "verified" here doesn't mean verified with Github, it means "verified as academic". We use a list of email domain patterns which are known to be academic. However, we certainly miss some, especially if they don't follow regular patterns. -- That's why we say to give details here about the domain address of your academic email and the institution it is associated with, so we can confirm academic status and add it manually. 

It looks like your public email on github is a `@icar.gov.in` address, which looks to be associated with the Indian Council of Agricultural Research. While not a traditional academic institution _per se_, it does likely fall under the general noncommercial use for ROSIE, so @lyskov should be able to added it to the approved list.

**Comment by lyskov** [comment, marked as answer]
This should be fixed now, - Could you please try to login again? Also, please feel free to re-open this issue if you still have trouble login-in. Thanks,

**Comment by Kutub-create** [comment]
?Perfect. Thank you very much! I can now log in..



????,

??????????? ??? ??????
????????? (???? ???????????????)
??? ?????? ??????

????????-????????? ???? ???????? ???????, ???-753006 ?????


With regards from,
Kutubuddin Ali Molla
Scientist (Biotechnology),
Crop Improvement Division,
ICAR-National Rice Research Institute,
Cuttack-753006, Odisha, India

Fulbright Postdoc Scholar (2017-2019)
Pennsylvania State University, USA-16801

Assistant Features Editor, The Plant Cell
INSA Young Scientist    ISCA Young Scientist
Member, Indian National Young Academy of Sciences (INYAS), New Delhi
________________________________
